src/main/java/seedu/address/model/deck/Deck.java

```diff
@@ -0,0 +1,85 @@
+package seedu.address.model.deck;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Objects;
+
+/**
+ * A group of cards
+ */
+public class Deck {
```

Comment:
It seemed @unfazing and @ruiyigan preferred that the Tag be morphed into Deck instead of creating a new Deck... I hope there is not too many refractoring to do if we create a new Deck class here.